README.md

```diff
@@ -35,4 +38,98 @@ nearup stop
+	Mar 25 01:39:50.807  INFO stats: #       0 Downloading headers 5% -/4  5/5/40 peers ⬇ 421.9kiB/s ⬆ 1.1kiB/s 0.00 bps 0 gas/s CPU: 47%, Mem: 75.3 MiB    
+	Mar 25 01:40:00.810  INFO stats: #       0 Downloading headers 6% -/4  5/5/40 peers ⬇ 508.3kiB/s ⬆ 1.2kiB/s 0.00 bps 0 gas/s CPU: 48%, Mem: 81.7 MiB    
+	Mar 25 01:40:10.810  INFO stats: #       0 Downloading headers 7% -/4  5/5/40 peers ⬇ 479.8kiB/s ⬆ 0.8kiB/s 0.00 bps 0 gas/s CPU: 33%, Mem: 86.4 MiB    
+	Mar 25 01:40:20.814  INFO stats: #       0 Downloading headers 7% -/4  5/5/40 peers ⬇ 442.0kiB/s ⬆ 0.4kiB/s 0.00 bps 0 gas/s CPU: 33%, Mem: 91.0 MiB    
+	Mar 25 01:40:31.570  INFO stats: #       0 Downloading headers 8% -/4  5/5/40 peers ⬇ 394.8kiB/s ⬆ 0.4kiB/s 0.00 bps 0 gas/s CPU: 38%, Mem: 96.4 MiB    
+	Mar 25 01:40:42.122  INFO stats: #       0 Downloading headers 9% -/4  5/5/40 peers ⬇ 348.0kiB/s ⬆ 0.4kiB/s 0.00 bps 0 gas/s CPU: 41%, Mem: 101.8 MiB    
+	Mar 25 01:40:52.124  INFO stats: #       0 Downloading headers 9% -/4  5/5/40 peers ⬇ 329.1kiB/s ⬆ 0.4kiB/s 0.00 bps 0 gas/s CPU: 49%, Mem: 107.5 MiB    
+	Mar 25 01:41:03.005  INFO stats: #       0 Downloading headers 10% -/4  5/5/40 peers ⬇ 319.7kiB/s ⬆ 0.3kiB/s 0.00 bps 0 gas/s CPU: 52%, Mem: 113.7 MiB    
+	Mar 25 01:41:13.008  INFO stats: #       0 Downloading headers 11% -/4  5/5/40 peers ⬇ 348.1kiB/s ⬆ 0.3kiB/s 0.00 bps 0 gas/s CPU: 59%, Mem: 120.1 MiB    
+	Mar 25 01:41:23.013  INFO stats: #       0 Downloading headers 12% -/4  5/5/40 peers ⬇ 358.0kiB/s ⬆ 0.4kiB/s 0.00 bps 0 gas/s CPU: 51%, Mem: 125.8 MiB
+	```
+
+### Cleaning up:
+In order to remove NEAR Betanet node and reclaim disk space, you have to:
+
+1. Stop nearup `nearup stop`:
+	```
+	Nearkats-MacBook-Pro:.nearup nearkat$ ./nearup stop
+	Stopping docker near
+	```
+2. Be sure to open the correct directory: `cd $HOME`
+3. Reclaim disk space by removing .near folder with the command `rm -ri .near`:
+	```
+	Nearkats-MacBook-Pro:.nearup nearkat$ rm -ri $HOME/.near
+	examine files in directory /Users/nearkat/.near? yes
+	remove /Users/nearkat/.near/genesis.json? yes
+	remove /Users/nearkat/.near/config.json? yes
+	remove /Users/nearkat/.near/node_key.json? yes
+	examine files in directory /Users/nearkat/.near/data? yes
+	remove /Users/nearkat/.near/data/IDENTITY? y
+	remove /Users/nearkat/.near/data/000003.log? y
+	remove /Users/nearkat/.near/data/LOCK? y
+	remove /Users/nearkat/.near/data/OPTIONS-000078? y
+	remove /Users/nearkat/.near/data/CURRENT? y
+	remove /Users/nearkat/.near/data/LOG? y
+	remove /Users/nearkat/.near/data/MANIFEST-000004? y
+	remove /Users/nearkat/.near/data/OPTIONS-000080? y
+	remove /Users/nearkat/.near/data? y
+	remove /Users/nearkat/.near/validator_key.json? y
+	remove /Users/nearkat/.near? y
+	```
+	You may save the LOG directory for future use. As a faster alternative, you can use `rm -rf .near`
+4. Remove `Docker`, by simply moving it from your applications folder to the trash
```

Comment:
I recall that you need to go to the Docker settings and clean the storage first as otherwise, it will keep those cached Docker images. You need to double-check that since I don't use macOS.